atom/browser/api/atom_api_menu_mac.mm

```diff
@@ -20,6 +20,8 @@
 
 namespace api {
 
+static NSMenu* applicationMenu_ = nil;
```

Comment:
Why does this have to be static?

Reply:
@MarshallOfSound With the `static` keyword, `applicationMenu_` remains visible only to this translation unit when the file is compiled.

Reply:
shouldn't it be in an unnamed namespace then? That's what we use in other obj-c files, e.g.:

```text
./atom/browser/ui/drag_util_mac.mm:namespace {
./atom/browser/ui/tray_icon_cocoa.mm:namespace {
./atom/browser/ui/file_dialog_mac.mm:namespace {
./atom/browser/ui/message_box_mac.mm:namespace {
./atom/browser/auto_updater_mac.mm:namespace {
./atom/browser/auto_updater_mac.mm:namespace {
./atom/browser/api/atom_api_browser_window_mac.mm:namespace {
./atom/browser/api/atom_api_system_preferences_mac.mm:namespace {
./atom/browser/mac/atom_application.mm:namespace {
./atom/browser/mac/in_app_purchase_observer.mm:namespace {
./atom/common/platform_util_mac.mm:namespace {
./atom/app/atom_main_delegate_mac.mm:namespace {
```

Reply:
@ckerr 👍 I've updated the code style